### PR TITLE
[bugfix](runtime-filter)  do not make query fail when rf publish failed 

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes

do not make query fail when rf publish failed 

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/exprs/runtime_filter_slots.h [hunk 1]
```diff
@@ -220,7 +220,11 @@ class RuntimeFilterSlotsBase {
     Status publish() {
         for (auto& pair : _runtime_filters) {
             for (auto filter : pair.second) {
-                RETURN_IF_ERROR(filter->publish());
+                auto state = filter->publish();
```
Comment (yiguolei) [inline be/src/exprs/runtime_filter_slots.h:223]: use status  not state

File: be/src/exprs/runtime_filter_slots.h [hunk 2]
```diff
@@ -220,7 +220,11 @@ class RuntimeFilterSlotsBase {
...
+                if (!state) {
+                    // TODO: solve publish failed when scan not schedured
+                    LOG(WARNING) << "filter publish failed, reason=" << state.to_string();
```
Comment (yiguolei) [inline be/src/exprs/runtime_filter_slots.h:226]: If state is not ok, why not return error status?

**Comment by BiteTheDDDDt** [comment]
run p0

File: be/src/exprs/runtime_filter_slots.h (hunk 2 above)
Comment (BiteTheDDDDt) [reply be/src/exprs/runtime_filter_slots.h:226]: Even if rf publish failed, the query should execute normally.
I changed this behavior in #20593 to cause the workflow to be blocked, now change it back

File: be/src/exprs/runtime_filter_slots.h (hunk 1 above)
Comment (BiteTheDDDDt) [reply be/src/exprs/runtime_filter_slots.h:223]: This piece of code will be removed in the future, so fix the workflow as soon as possible

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**